cpp/src/Glacier2/FilterManager.h

```diff
@@ -51,4 +51,4 @@ namespace Glacier2
     };
 }
 
-#endif /* FILTER_MANAGER_H */
```

Comment:
Same idea. I'm going to stop pointing them out, I'm sure you get the idea.
Anyways, I'd just leave these normal block comments alone, and only focus on the block doc-comments for now.